geoengine/workflow.py

```diff
@@ -534,6 +535,44 @@ def save_as_dataset(
             upload_id=UploadId(response_json['upload'])
         )
 
+    def create_save_as_dataset_task(
+            self,
+            bbox: QueryRectangle,
+            name: str,
+            description: str = '',
+            timeout: int = 3600) -> TaskId:
+        '''EXPERIMENTAL: Init task to store the workflow result as a layer'''
```

Comment:
What makes this more experimental as everything else? :smile:

Reply:
Idk. I removed it.